chaosazure/__init__.py

```diff
@@ -74,14 +106,38 @@ def discover(discover_system: bool = True) -> Discovery:
     return discovery
 
 
+def init_client(secrets, configuration) -> ComputeManagementClient:
```

Comment:
Thanks for setting the type of the returned value, could you also set the typing annotation on the parameters please?